pipelines/operator-hosted-pipeline.yml

```diff
@@ -0,0 +1,255 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: operator-hosted-pipeline
+spec:
+  params:
+    - name: git_pr_branch
+    - name: git_pr_title
+    - name: git_pr_url
+    - name: git_repo_url
+    - name: git_username
+    - name: pyxis_url
+    - name: bundle_path
+    - name: preflight_latest_version
+    - name: ci_latest_version
+    - name: internal_registry_url
+      default: default-route-openshift-image-registry.apps-crc.testing
+    - name: quay_url
+      default: quay.io
+  workspaces:
+    - name: repository
+    - name: results
+  tasks:
+    # Git clone
+    - name: checkout
+      taskRef:
+        name: git-clone
+        kind: ClusterTask
+      params:
+        - name: url
+          value: $(params.git_repo_url)
+        - name: revision
+          value: $(params.git_pr_branch)
+      workspaces:
+        - name: output
+          workspace: repository
+          subPath: src
+
+    # Validate submission
+    - name: submission-validation
+      runAfter:
+        - checkout
+      taskRef:
+        name: submission-validation
+      params:
+        - name: git_username
+          value: $(params.git_username)
+        - name: git_pr_title
+          value: $(params.git_pr_title)
+        - name: pyxis_url
+          value: $(params.pyxis_url)
+      workspaces:
+        - name: source
+          workspace: repository
+          subPath: src
+
+    # reserve operator name
+    - name: reserve-operator-name
+      runAfter:
+        - submission-validation
+      taskRef:
+        name: reserve-operator-name
+      params:
+        - name: operator_name
+          value: "$(tasks.submission-validation.results.operator_name)"
+
+    # additional checks
+    - name: operator-validation
+      runAfter:
+        - reserve-operator-name
+      taskRef:
+        name: operator-validation
+      params:
+        - name: bundle_path
+          value: "$(params.bundle_path)"
+      workspaces:
+        - name: source
+          workspace: repository
+          subPath: src
+
+    - name: yaml-lint
+      runAfter:
+        - reserve-operator-name
+      taskRef:
+        name: yaml-lint
+      params:
+        - name: args
+          value: ["-d {extends: relaxed, rules: {line-length: {max: 180}}}", "$(params.bundle_path)"]
```

Comment:
Let's use the same rules as in CI pipeline: https://github.com/redhat-openshift-ecosystem/operator-pipelines/blob/main/pipelines/operator-ci-pipeline.yml#L67

Relaxed profile validation is very limited and many things pass.